Jeplind Morico Gugundo

jeplindm opened the pull request:
layout sign in
layout sign up
layout kanban board

Commit:
layout sign in
layout sign up
layout kanban board

Commit:
rev. kanban board added edit

Commit:
rev. kanban board added edit+delete button

Commit:
Commit Server
add migration user
add migration task
create database development
validate model user
validate model task
set up app.js
create routes index.js
create route /register
create user-controller
create middlewares error-handler
route register succcess
create helpers bcrypt
hash password with user add hook beforecreate

Commit:
fixing layout and added add-task form

Commit:
1. add server: login
2. add helpers: jwt
3. add routes: /login

Commit:
server done

Commit:
client
client login with server
client regsiter with server

Commit:
fetch data client with server

Commit:
fetch + add task + edit task client using server

Commit:
delete task by client with server

Commit:
fetch + add task + edit task + patch task(draggable) + google login

Commit:
set up app js, config, package.json

Commit:
fix config.json

Commit:
Kanban Portofolio Final
1. Deploy server to heroku
2. Deploy client to firebase
3. Google login
4. API Documentation
5. Done

Commit:
Insert url deploy in readme.md

jeplindm commented:
.env
PORT=3015
SALT=10
SECRET=bigsecret

dhimashary commented on client/src/App.vue:
```diff
@@ -0,0 +1,379 @@
+                method: 'POST',
+                data:  { google_access_token }
+            })
+            .then(data => {
```
ini bisa di destructure aja ({data}) biar dibawahnya gak 
data.data

dhimashary commented on client/src/App.vue:
```diff
@@ -0,0 +1,379 @@
+            })
+        },
+        logout(clear) {
+            console.log('logout sukses');
```
console log kalo udah deploy diapus ya

dhimashary commented on client/src/App.vue:
```diff
@@ -0,0 +1,379 @@
+        },
+        logout(clear) {
+            console.log('logout sukses');
+            clear
```
clear?

dhimashary commented on client/src/components/NavbarPage.vue:
```diff
@@ -0,0 +1,111 @@
+            let clear = localStorage.clear()
+            this.$emit('clearStorage', clear)
+        },
+        changeToRegister() {
```
kenapa ga dibuat aja jadi 1 ? misal changePage(page)
ini banyak banget function yang gunanya sama

dhimashary commented:
edit delete punya orang lain baiknya di hide aja

dhimashary commented:
delete baiknya ada konfirmasi apakah yakin mau delete

dhimashary commented:
kotak categorynya sempit banget, bisa agak di gedein? terus baiknya ada pading kiri kanan biar tasknya ga nempel banget

dhimashary commented:
.env kok ga di ignore

jeplindm commented:
> .env kok ga di ignore

Commit:
deleted .env from repo

Commit:
Revisi Kanban
1. git ignore .env
2. destructure data in google sign in method
3. deleted console.log
4. fixing sign out method
5. makin change page method efficiently
6. adding delete feature with confirmation
7. fixing category and task card

Commit:
re-deploy
